compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -447,22 +447,27 @@ execIde telemetry (Debug debug) enableScenarioService options =
             -- performance will be significatly impacted by large log output.
             threshold
             "LanguageServer"
+          damlCacheM <- lookupEnv "DAML_CACHE"
+          damlHomeM <- lookupEnv "DAML_HOME"
+          fallback <- getXdgDirectory XdgCache "daml"
+          let cacheDir = fromMaybe fallback $ damlCacheM <|> damlHomeM
           let gcpConfig = Logger.GCP.GCPConfig
                   { gcpConfigTag = "ide"
-                  , gcpConfigDamlPath = Nothing
+                  , gcpConfigCachePath = cacheDir
                   }
               withLogger f = case telemetry of
```

Comment:
Afaict, we only need to set $HOME everywhere because `getXdgDirectory` needs it. Can we just disable telemetry if `getXdgDirectory` fails instead? I’m slightly worried users might hit this in CI as well.

Reply:
changed, I got really annoyed of setting $HOME everywhere ...

Reply:
So this means running damlc without having `DAML_HOME` set won’t enable telemetry? That seems perfectly acceptable to me :+1: